cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/constants/WebServerConfig.java

```diff
@@ -114,21 +114,6 @@
   public static final String WEBSERVER_ACCESSLOG_ENABLED_CONFIG = "webserver.accesslog.enabled";
   public static final boolean DEFAULT_WEBSERVER_ACCESSLOG_ENABLED = true;
   public static final String WEBSERVER_ACCESSLOG_ENABLED_DOC = "true if access log is enabled";
-
-  /**
-   * <code>webserver.accesslog.path</code>
-   */
-  public static final String WEBSERVER_ACCESSLOG_PATH_CONFIG = "webserver.accesslog.path";
-  public static final String DEFAULT_WEBSERVER_ACCESSLOG_PATH = "access.log";
-  public static final String WEBSERVER_ACCESSLOG_PATH_DOC = "HTTP Request log path";
-
-  /**
-   * <code>webserver.accesslog.retention.days</code>
-   */
-  public static final String WEBSERVER_ACCESSLOG_RETENTION_DAYS_CONFIG = "webserver.accesslog.retention.days";
-  public static final int DEFAULT_WEBSERVER_ACCESSLOG_RETENTION_DAYS = 7;
-  public static final String WEBSERVER_ACCESSLOG_RETENTION_DAYS_DOC = "HTTP Request log retention days";
-
```

Comment:
We should ideally keep these configs and mark them as deprecated to be removed in a future release. Or we should ensure that the corresponding references to these configs should be removed from Configurations.md and cruisecontrol.properties files.

Reply:
Hi @amuraru Are you still working on this PR?

Reply:
thanks for your reminder @efeg - let me review this in the upcoming days.

Reply:
@efeg Removed  corresponding references to these configs from Configurations.md and cruisecontrol.properties files.

Let me know if this looks good to you. I felt we shouldn't go through a deprecation,removal cycle for these